jerry-core/ecma/operations/ecma-objects.c

```diff
@@ -3261,6 +3264,78 @@ ecma_op_ordinary_object_has_own_property (ecma_object_t *object_p, /**< the obje
   return property != ECMA_PROPERTY_TYPE_NOT_FOUND && property != ECMA_PROPERTY_TYPE_NOT_FOUND_AND_STOP;
 } /* ecma_op_ordinary_object_has_own_property */
 
+#if (JERRY_ESNEXT)
```

Comment:
Why not JERRY_BUILTIN_WEAKREF

Reply:
If you check it more thoroughly, this method is used by weakmap/weaksets as well. So anyway the remaks it reasonable, however the correct guard would be :

```
#if JERRY_BUILTIN_WEAKREF || JERRY_BUILTIN_WEAKSET || JERRY_BUILTIN_WEAKMAP
```